internal/output/table_factory.go

```diff
@@ -97,7 +114,14 @@ func (tf *TableFactory) SetColumnFormatter(column string, columnFormatter func(i
 }
 
 // ToTable creates a tablewriter.Table instance with sorted data and default rendering settings
-func (tf *TableFactory) ToTable() *tablewriter.Table {
+func (tf *TableFactory) ToTable() (*tablewriter.Table, error) {
+	if tf.exportFile != "" {
+		return tf.newCsvTable()
```

Comment:
Shouldn't that depend somehow on the exportFormat set?

Reply:
Oh yeah for sure. Done 👍